api/PerformanceElementTiming.json

```diff
@@ -311,9 +311,16 @@
             "web-features:element-timing"
           ],
           "support": {
-            "chrome": {
-              "version_added": "77"
-            },
+            "chrome": [
+              {
+                "version_added": "133"
+              },
+              {
+                "version_added": "77",
+                "partial_implementation": true,
+                "notes": "Returns 0 for cross-origin media."
+              }
```

Comment:
Even if it seems verbose, I think it would be better to add a subfeature here as well.

/cc @Elchi3

Reply:
Ah yes you are correct. Missed doing that here. Will add.

Reply:
Updated. PTAL.